integration_tests/src/main/python/date_time_test.py

```diff
@@ -278,6 +278,13 @@ def test_from_utc_timestamp_unsupported_timezone_fallback(data_gen, time_zone):
     'FromUTCTimestamp')
 
 
+@pytest.mark.parametrize('time_zone', ["UTC", "America/Los_Angeles", "Asia/Shanghai"], ids=idfn)
```

Comment:
Phase 1, we only support non-repeat time zone.
`America/Los_Angeles` will not work.